specificatie/domain.yaml

```diff
@@ -745,6 +749,9 @@ components:
     PerceelBasis:
       allOf:
         - $ref: "#/components/schemas/KadastraalOnroerendeZaakBasis"
+        - properties:
+            indicatieSluimerend:
```

Comment:
@kad-pothot @kad-hebbim: ik dacht dat alleen appartementsrechten sluimerend konden zijn (percelen niet). Deze indicatieSluimerend komt dan te vervallen. Is dat juist?

Reply:
Klopt inderdaad.

Reply:
Aangepast